templates/chrony.keys.epp

```diff
@@ -1,5 +1,5 @@
 <% if $chrony::chrony_password and $chrony::chrony_password != 'unset' { -%>
```

Comment:
I don't think it needs unwrapping here, but the line does look a little broken?  Given its type,  I don't think `$chrony::chrony_password` can ever be `falsey`?

Optionally, feel free to fix that, (and/or parameterize this template) ;)

Reply:
I've reworked the template to be smarter.